integrations/friendbuy/HISTORY.md

```diff
@@ -1,3 +1,9 @@
+1.1.0 / 2020-12-11
```

Comment:
@mericsson apologies—I belatedly realized that I should be the one to change the version 😅 I have now modified it along with the date. Let me know if any further issues arise. Thanks!